### PR TITLE
User migration procedure: clarify number of accounts requirement

### PR DESCRIPTION
**Pull request by zuluecho9**
User mgmt liaison work: some slight clarification around # of accounts requirement for user migration.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-saml-scim.mdx [hunk 1]
```diff
@@ -5,14 +5,14 @@ redirects:
   - /docs/okta-scim-assign-users-automated-provisioning-beta
 ---
 
-For setting up automatic controls for how your New Relic users are added and managed, and how they log in, we offer these features: 
+For setting up automatic controls for how your New Relic users are added to New Relic, how they're managed, and how they log in, we offer these features:
```
Comment (bradleycamacho) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-saml-scim.mdx:8]:
```suggestion
For setting up automatic controls over how your New Relic users are added to New Relic, how they're managed, and how they log in, we offer these features: 
```

Comment (bradleycamacho) [reply src/content/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-saml-scim.mdx:8]: I stumbled reading the first part of the sentence.

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 2]
```diff
@@ -10,7 +10,7 @@ Starting April 12, 2021, we're allowing some customers who have users on our ori
...
 
-At first, this new model was available mainly to new customers, and users in pre-existing New Relic organizations remained on our original user model. But now some original-model organizations that meet [requirements](#requirements) can use a migration wizard to convert their users to the new model. When the migration process is complete, your users will be on the New Relic One user model and you’ll have [new procedures for managing your users](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/).  
+At first, this new model was available mainly to new customers, while users in pre-existing New Relic organizations remained on our original user model. But now some original-user-model organizations that meet some [requirements](#requirements) can use a migration wizard to migrate their users to the new model. When that migration process is complete, your users are on the New Relic One user model and you’ll have [new procedures for managing your users and their access to accounts](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/).
```
Comment (bradleycamacho) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:13]: Is there a doc link to the wizard they can use?

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 3]
```diff
@@ -26,18 +26,17 @@ When you migrate your users to this model, benefits include:
...
-* If you frequently [add accounts](/docs/accounts/accounts-billing/account-structure/mastersub-account-structure/#creating) to your organization, we recommend **not** using the wizard at this point in time. Currently, with the new model, you must get New Relic assistance to add accounts. 
-* Roles related to user management (ability to add and update users, change user type, create access grants) are currently applied for an entire organization and can't be assigned at the account level. If you require account-level user management role assignment, we recommend not using the wizard. 
-* If you were previously using OneLogin, Azure, or Okta apps for a New Relic integration, note that you'll likely need a different, newer app titled "New Relic by organization" (and not "New Relic by account").
+* If your organization has more than 20 accounts, we recommend **not** using the user migration wizard. **You can disregard this requirement if you're being helped by New Relic representatives.** (Note that this refers to having 20 or more accounts in your [organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#new-model), not users.)
```
Comment (bradleycamacho) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:34]: Is there a next step for orgs that have 20+ accounts, but no rep?

**Review by bradleycamacho — APPROVED** [review]
Left a few comments, looks good otherwise!

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx (hunk 3 above)
Comment (zuluecho9) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:34]: No, no next steps really. It kind of preselects for companies that have reps.

**Pull request merged**